pyproject.toml

```diff
@@ -0,0 +1,20 @@
+[tool.black]
+py36 = true
```

Comment:
Since this is the first project that (from anders side) that will use Black properly, I would like to go with only defaults for now and see if there are any caveats with doing so. So if you are ok with it, I would like to keep the default of 88.

Reply:
Fine by me. :+1: